recipes/dlpack/all/conanfile.py

```diff
@@ -0,0 +1,25 @@
+import os
+from conans import ConanFile, CMake, tools
+
```

Comment:
```suggestion

required_conan_version = ">=1.33.0"

```

[strip_root](https://docs.conan.io/en/latest/changelog.html?highlight=strip_root#jan-2021) requires Conan 1.33

Reply:
maybe, but anyway it's informational only, as CCI requires latest conan client already